includes/Modules/PageSpeed_Insights.php

```diff
@@ -129,17 +129,15 @@ protected function create_data_request( Data_Request $data ) {
 	 * Parses a response for the given datapoint.
 	 *
 	 * @since 1.0.0
+	 * @since n.e.x.t GET:pagespeed returns the full response, not just the lighthouse result.
 	 *
 	 * @param Data_Request $data Data request object.
 	 * @param mixed        $response Request response.
 	 *
 	 * @return mixed Parsed response data on success, or WP_Error on failure.
 	 */
 	protected function parse_data_response( Data_Request $data, $response ) {
-		switch ( "{$data->method}:{$data->datapoint}" ) {
-			case 'GET:pagespeed':
-				// TODO: Parse this response to a regular array.
-				return $response->getLighthouseResult();
+		switch ( "{$data->method}:{$data->datapoint}" ) { // phpcs:ignore Generic.CodeAnalysis.EmptyStatement.DetectedSwitch
 		}
```

Comment:
We could just remove this I guess?

Reply:
I figure we'll add something to it in the future and the case was the only thing really to remove. We can remove the whole switch if you'd like.